App.js

```diff
@@ -61,7 +61,14 @@ class App extends Component {
       {
         storage: AsyncStorage,
         transforms: [encryptor],
-        blacklist: ['user'],
+        blacklist: [
+          'user',
+          'issue',
+          'repository',
+          'organization',
+          'search',
+          'notifications',
+        ],
```

Comment:
@machour Setting `whitelist: ['auth']` can be simpler.

Reply:
And if history search depends on `search`, we should also add it to the whitelist. I am not sure.

Reply:
Do we have a search history? Oo
Looking into `whitelist` now :)

Reply:
Fixed to use `whitelist`, thank you for the hint @chinesedfan!
Kept `search` blacklisted as it currently only holds only the result set from the API, and there's no history yet.

Reply:
@machour Thanks for your investigation. #55 mentions historical search results but seems not have implemented yet.